Packages/Sources/ConnectWalletFeature/ConnectWalletFeature.swift

```diff
@@ -219,38 +219,37 @@ struct PopUpModifier: ViewModifier {
             label: {
               Text("Open")
                 .font(Font(FontName.dmSansBold, size: 14))
-                .frame(maxWidth: .infinity)
+                .frame(minWidth: 160)
```

Comment:
The popup does not look good on smaller devices, it is too big. Maybe better way would be start from the outer margins. See this pic from Figma 
![image](https://github.com/hilalalhakani/NodeCrypto/assets/1016974/7f29b746-5e7b-46d6-9e2c-f589c58ad72c)

It can be interpreted like: the popup is 32 from system margins, inner text padding is 32. Buttons have fixed width of 160. Maybe it will look better